src/tests/ftest/util/server_utils.py

```diff
@@ -293,7 +293,7 @@ def prepare_storage(self, user, using_dcpm=None, using_nvme=None):
             cmd.sub_command_class.sub_command_class.nvme_only.value = True
 
         self.log.info("Preparing DAOS server storage: %s", str(cmd))
-        results = run_pcmd(self._hosts, str(cmd), timeout=40)
+        results = run_pcmd(self._hosts, str(cmd), timeout=self.prepare_timeout)
```

Comment:
(pylint-no-member) Instance of 'DaosServerManager' has no 'prepare_timeout' member